core/core_modules/disable.js

```diff
@@ -1,24 +1,117 @@
-var disabled = false;
+var prevTimeStamp;
+var counter = 0,
+var commands = [
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['var' is defined but never used.](https://www.codacy.com/app/theprojectshop/Kassy/file/2554872520/issues/source?bid=3282987&fileBranchId=3282995#l3)